docs/layers.md

```diff
@@ -665,30 +665,57 @@ At zoom 12 and below, we consolidate some landuse kinds to reduce the amount of
 
 The current mappings are:
 
+* `airfield` -> `aerodrome`
 * `allotments` -> `urban_area`
+* `artwork` -> `urban_area`
+* `attraction` -> `urban_area`
 * `bare_rock` -> `desert`
-* `cemetery` -> `urban_area`
+* `cinema` -> `urban_area`
+* `college` -> `university`
 * `commercial` -> `urban_area`
+* `common` -> `grassland`
+* `container_terminal` -> `urban_area`
+* `cutline` -> `forest`
+* `dam` -> `land`
+* `danger_area` -> `military`
 * `farm` -> `farmland`
+* `ferry_terminal` -> `urban_area`
+* `fort` -> `urban_area`
+* `generator` -> `urban_area`
 * `grass` -> `grassland`
 * `heath` -> `grassland`
 * `industrial` -> `urban_area`
 * `meadow` -> `grassland`
 * `mud` -> `wetland`
 * `natural_wood` -> `forest`
 * `orchard` -> `farmland`
+* `pitch` -> `urban_area`
+* `place_of_worship` -> `urban_area`
+* `plant` -> `urban_area`
 * `plant_nursery` -> `farmland`
+* `port` -> `urban_area`
+* `port_terminal` -> `urban_area`
+* `prison` -> `urban_area`
 * `quarry` -> `barren`
+* `railway` -> `urban_area`
+* `range` -> `military`
 * `recreation_ground` -> `urban_area`
 * `residential` -> `urban_area`
 * `retail` -> `urban_area`
+* `rock` -> `barren`
 * `sand` -> `desert`
 * `school` -> `urban_area`
 * `scree` -> `barren`
+* `scrub` -> `grassland`
+* `service_area` -> `urban_area`
```

Comment:
If we don't merge `rest_area`, then we shouldn't merge `service_area` either. I can see remapping to `rest_area`, though.

Reply:
Good point. I'm not sure a `service_area` and `rest_area` are likely to get merged, though? Do we often find two adjacent landuse polygons of these different types?

For the time being, I've removed the remapping of `service_area` in bc5de76.